packages/test/test-service-load/src/utils.ts

```diff
@@ -126,12 +126,16 @@ class MockDetachedBlobStorage implements IDetachedBlobStorage {
     }
 }
 
-export async function initialize(testDriver: ITestDriver, seed: number, testConfig: ILoadTestConfig,
-    verbose: boolean, testIdn?: string) {
+export async function initialize(
+    testDriver: ITestDriver,
+    seed: number,
+    endpoint: DriverEndpoint | undefined,
+    testConfig: ILoadTestConfig,
+    verbose: boolean, testIdn?: string,
+) {
     const randEng = random.engines.mt19937();
     randEng.seed(seed);
-    const optionsOverride =
```

Comment:
@anthony-murphy I hope it wasn't deliberate that this code uses testDriver.endpointName whereas the code in runner.ts uses endPoint passed to it.